paper-toast.html

```diff
@@ -117,23 +136,34 @@
      * @method toggle
      */
     toggle: function() {
-      this.opened = !this.opened;
+      if (!this.visible) {
+        this.show();
+      } else {
+        this.hide();
+      }
     },
 
-    /**
-     * Show the toast.
-     * @method show
-     */
-    show: function() {
-      this.opened = true;
-      if (PaperToast.currentToast && PaperToast.currentToast !== this) {
-        PaperToast.currentToast.hide();
-      }
-      PaperToast.currentToast = this;
-      this._toggleClasses(true);
-      this.debounce('dismissToast', this.hide, this.duration);
+    _visibleChanged: function(visible) {
+      this.toggleClass('paper-toast-open', visible);
+    },
+
+    _ariaAnnounce: function() {
+      var oldText = this.text;
+
+      this.setAttribute('aria-live', 'assertive');
+      this.$.label.setAttribute('role', 'alert');
+      this.$.label.setAttribute('aria-live', 'assertive');
+
+      this.text = '';
```

Comment:
I'm sure it's on purpose, but why do you need to empty the first and then set it correctly?